pip/qsharp/_qsharp.py

```diff
@@ -343,6 +343,9 @@ def __str__(self) -> str:
     def _repr_html_(self) -> str:
         return self.__data._repr_html_()
 
+    def _repr_latex_(self) -> str:
+        return self.__data._repr_latex_()
```

Comment:
Make sure that there's something displayed when LaTeX representation isn't available.